amfora.go

```diff
@@ -68,10 +70,34 @@ func main() {
 	display.NewTab()
 	if len(os.Args[1:]) > 0 {
 		display.URL(os.Args[1])
+	} else if !isStdinEmpty() {
+		renderFromStdin()
 	}
 
 	// Start
 	if err = display.App.Run(); err != nil {
 		panic(err)
 	}
 }
+
+func isStdinEmpty() bool {
+	stat, _ := os.Stdin.Stat()
+	return (stat.Mode() & os.ModeCharDevice) != 0
+}
+
+func renderFromStdin() {
```

Comment:
Why not just use `io.Copy` here? It would be a lot more efficient. And then keep it as bytes until the last moment you need to change it, in `renderPageFromString`.